index.js

```diff
@@ -1204,11 +1202,14 @@ function nested (laterCode, locationPath, key, location, subKey, isArray) {
           // 2. `nested`, through `buildCode`, replaces any reference in object properties with the actual schema
           // (see https://github.com/fastify/fast-json-stringify/blob/6da3b3e8ac24b1ca5578223adedb4083b7adf8db/index.js#L631)
 
+          const extendedSchema = clone(location.schema)
```

Comment:
Can you add a comment here to let other know why this approach is a must (like the previous one). Thanks.